client/src/App.js

```diff
@@ -1,116 +1,185 @@
-import { Switch, Route } from 'react-router-dom';
-import React, { useState, useEffect } from 'react';
-import axios from 'axios';
 import styled from 'styled-components';
-import { useLocalStorage } from '../src/hooks/useLocalStorage';
+import { useState, useEffect } from 'react';
+import { Switch, Route } from 'react-router-dom';
 import Home from './Pages/Home';
-import Watchlist from './Pages/Watchlist';
-import Friends from './Pages/Friends';
 import Search from './Pages/Search';
+import axios from './services/axios';
+import Friends from './Pages/Friends';
+import Ratings from './Pages/Ratings';
+import Profile from './Pages/Profile';
+import Watchlist from './Pages/Watchlist';
+import Favorites from './Pages/Favorites';
+import Sidebar from './components/Sidebar';
+import requests from './services/requests';
 import Banner from '../src/components/Banner';
-import Navigation from '../src/components/Navigation';
-import Store from './Store';
 import Searchbar from './components/Searchbar';
+import FriendsCards from './Pages/FriendsCards';
+import ScrollToTop from './services/ScrollToTop';
+import Navigation from '../src/components/Navigation';
+import { useLocalStorage } from '../src/hooks/useLocalStorage';
```

Comment:
Maybe you could group the imports: 
- `./Pages/...`
- `./Components/...`
- `./services/...`
- `./hooks/...`

Reply:
You are right, makes more sense! :)